demo.md

```diff
@@ -0,0 +1,23 @@
+## Demo script:
+
+- ERC-4337 compatible Smart Contract wallet
+- Unique feature: gasless off-chain txs with instant finality enabled by HTLC technology, as used in Bitcoin Lightning. Funds can be moved in and out of the off-chain payment system without any on-chain txs!
+- Setup required:
+  - on-chain: wallet owners must designate an Intermediary in their SC wallet
+  - off-chain: wallet owners and Intermediary must sign-off on some amount of wallet funds designated for off-chain txs
+  - run in-browser client(s) to interact with SC wallet and peers
+  - `Explain browser tab setup with Alice(left), Irene(middle), Bob(right)`
+- How it works:
+  - on-chain txs are 2/2 multi-sig (user + intermediary)
+  - `Show on-chain tx from Alice to Metamask account`
```

Comment:
It might be nice to do this on the receiving end, and have Bob show the transaction. We could have a flow like this:
- Alice and Bob start with a balance of [5,5]
- Alice pays Bob (crosschain!) an amount of 3
- We see all the balances update 
- Bob now has a balance of 8
- Bob sends an L1 transaction of 7 (which he couldn't afford before the payment) to some external address.

Reply:
Good call. That'd be a good way to show the cross-chain functionality

Reply:
Yes I like this!